code/modules/mob/living/carbon/carbon.dm

```diff
@@ -690,6 +690,9 @@
 		spawn(time)
 			make_visible(source_define)
 
+/mob/living/carbon
+	var/ice_sliding = 0
```

Comment:
do we really not have an existing code block where vars are set for carbon?

Reply:
I think most of those vars are set on carbon/human or carbon/alien for humanoids and benos respectively.

Reply:
I think pedr poses a valid point in that we are starting to have so many slide effects that at some point we might prefer to have a single slide proc like we have a throw proc

Reply:
I can add a generic "slide" proc and shift some things to use it. Depends on how "atomic" you want this PR to be though. I have no problem making a ton of different changes in this one PR but I read that veegee likes PR's to only change one thing at a time. If the concern is that I won't fix any bugs that arise, well, I will.

Reply:
it's fine to make helpers generally speaking

Reply:
actually what i meant was adding vars to a type, usually that's done right at the top of the file
carbon.dm seems to have only an admin_desc and this being updated, and i think it's weird that you put this 600 lines in instead of the top